lib/resolvers/admin_mutations/admin-remove-group-chat.js

```diff
@@ -4,14 +4,17 @@ const adminCheck = require('../functions/adminCheck');
 const { NotFoundError } = require('errors');
 const requestContext = require('talawa-request-context');
 
-const Group = require('../../models/Group');
+const GroupChat = require('../../models/GroupChat');
 
 module.exports = async (parent, args, context) => {
   //find message
-  let group = await Group.findOne({ _id: args.groupId });
+  let group = await GroupChat.findOne({ _id: args.groupId });
+
   if (!group) {
     throw new NotFoundError(
-      requestContext.translate('group.notFound'),
+      process.env.NODE_ENV !== 'production'
```

Comment:
utilize the variable constants in the constants file and remove the stringed variables.